Dockerfile

```diff
@@ -1,31 +1,21 @@
-FROM ruby:latest
+FROM registry.redhat.io/ubi8/ruby-25
```

Comment:
I would prefer to stay with a base image that can be pulled without authentication. Maybe we could add a `Dockerfile.ubi8` for when you want to build on top of the ubi image?